packages/features/bookings/Booker/components/AvailableTimeSlots.tsx

```diff
@@ -53,29 +61,18 @@ export const AvailableTimeSlots = ({ extraDays, limitHeight, seatsPerTimeSlot }:
   };
 
   const schedule = useScheduleForEvent({
-    prefetchNextMonth: !!extraDays && dayjs(date).month() !== dayjs(date).add(extraDays, "day").month(),
+    prefetchNextMonth,
+    monthCount,
   });
+  const nonEmptyScheduleDays = useNonEmptyScheduleDays(schedule?.data?.slots).filter((slot)=>dayjs(selectedDate).diff(slot,'day')<=0);
```

Comment:
```suggestion
  const somethingThatDescribesWhatFilterDoes = schedule?.data?.slots).filter((slot)=>dayjs(selectedDate).diff(slot,'day')<=0;
  const nonEmptyScheduleDays = useNonEmptyScheduleDays(somethingThatDescribesWhatFilterDoes);
```
Increasing script by 1 line but it will become more readable. 😄

Reply:
Basically this filters returns available dates from selected date till the end.